CHANGELOG.md

```diff
@@ -275,6 +275,81 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - `@cumulus/message/Queue.getQueueName`
   - `@cumulus/message/Queue.getQueueNameByUrl`
 
+## v2.0.2+ Backport releases
+
+Release v2.0.1 was the last release on the 2.0.x release series.
+
+Changes after this version on the 2.0.x release series are limited
+security/requested feature patches and will not be ported forward to future
+releases unless there is acorresponding CHANGELOG entry.
```

Comment:
```suggestion
releases unless there is a corresponding CHANGELOG entry.
```

Reply:
I really hate blocking merge of this PR on something like this, so I'm also happy if we want to just not worry about this one

Reply:
It's simple enough to fix and re-run, I don't think this PR's priority is such that we have to allow mistakes in.